src/Avalonia.Visuals/Point.cs

```diff
@@ -267,5 +267,24 @@ public Point WithY(double y)
         {
             return new Point(_x, y);
         }
+
+        /// <summary>
+        /// Deconstructor for decomposition Point
+        /// </summary>
+        /// <param name="x">The X position.</param>
```

Comment:
I'd prefer "coordinate" rather than "position" for consistency with other methods (e.g. `WithX`).

Reply:
But more place in code have "position"
![image](https://user-images.githubusercontent.com/29205560/94556421-b5347b00-0265-11eb-8a7d-9248253cdf8d.png)

Reply:
Oh yes, sorry.